airbyte-commons/src/main/resources/log4j2.xml

```diff
@@ -2,40 +2,69 @@
-        <!-- Remove paths from logs on Cloud as users aren't able to access file paths and these end up being noise. -->
-        <Property name="cloud-worker-file-pattern">%replace{%X{log_source} - }{^ - }{}%d{yyyy-MM-dd HH:mm:ss}{GMT+0} %p %C{1}(%M):%L - %replace{%m}{apikey=[\w\-]*}{apikey=*****}%n</Property>
+        <!-- Mask the string apikey=<string> to apikey=***** to prevent secrets leaking. -->
+        <Property name="default-pattern">%d{yyyy-MM-dd HH:mm:ss}{GMT+0} %highlight{%p} %C{1.}(%M):%L - %replace{%m}{apikey=[\w\-]*}{apikey=*****}%n</Property>
+        <!--Logs the timestamp and log_source/application name in the beginning of the line if it exists with a > separator, and then always the rest of the line.-->
+        <Property name="simple-pattern">%d{yyyy-MM-dd HH:mm:ss}{GMT+0}%replace{ %X{log_source}}{^ -}{} > %replace{%m}{apikey=[\w\-]*}{apikey=*****}%n</Property>
 
         <!-- Always log INFO by default. -->
-        <Property name="log-level">$${env:LOG_LEVEL:-INFO}</Property>
+        <Property name="log-level">$${sys:LOG_LEVEL:-${env:LOG_LEVEL:-INFO}}</Property>
 
-        <Property name="s3-bucket">$${env:S3_LOG_BUCKET}</Property>
-        <Property name="s3-region">$${env:S3_LOG_BUCKET_REGION}</Property>
-        <Property name="s3-aws-key">$${env:AWS_ACCESS_KEY_ID}</Property>
-        <Property name="s3-aws-secret">$${env:AWS_SECRET_ACCESS_KEY}</Property>
-        <Property name="s3-minio-endpoint">$${env:S3_MINIO_ENDPOINT}</Property>
-        <Property name="s3-path-style-access">$${env:S3_PATH_STYLE_ACCESS}</Property>
+        <Property name="s3-bucket">$${sys:S3_LOG_BUCKET:-${env:S3_LOG_BUCKET}}</Property>
+        <Property name="s3-region">$${sys:S3_LOG_BUCKET_REGION:-${env:S3_LOG_BUCKET_REGION}}</Property>
+        <Property name="s3-aws-key">$${sys:AWS_ACCESS_KEY_ID:-${env:AWS_ACCESS_KEY_ID}}</Property>
+        <Property name="s3-aws-secret">$${sys:AWS_SECRET_ACCESS_KEY:-${env:AWS_SECRET_ACCESS_KEY}}</Property>
+        <Property name="s3-minio-endpoint">$${sys:S3_MINIO_ENDPOINT:-${env:S3_MINIO_ENDPOINT}}</Property>
+        <Property name="s3-path-style-access">$${sys:S3_PATH_STYLE_ACCESS:-${env:S3_PATH_STYLE_ACCESS}}</Property>
 
-        <Property name="gcs-log-bucket">$${env:GCS_LOG_BUCKET}</Property>
+        <Property name="gcs-log-bucket">$${sys:GCS_LOG_BUCKET:-${env:GCS_LOG_BUCKET}}</Property>
     </Properties>
 
     <Appenders>
         <Console name="Default" target="SYSTEM_OUT">
+            <ContextMapFilter onMatch="DENY" onMismatch="ACCEPT">
+                <KeyValuePair key="simple" value="true"/>
+            </ContextMapFilter>
             <PatternLayout pattern="${default-pattern}"/>
         </Console>
 
+        <Console name="SimpleDefault" target="SYSTEM_OUT">
+            <ContextMapFilter onMatch="ACCEPT" onMismatch="DENY">
```

Comment:
what does the `ContextMapFilter` do?

Reply:
It looks at the context map and decides if it should use this logger or not. So throughout the file we're deciding whether to use simple or full logging.

Reply:
got it, thanks!